samples/create-training-pipeline-image-classification.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main(
+  datasetId,
+  modelDisplayName,
+  trainingPipelineDisplayName,
+  project,
+  location = 'us-central1'
+) {
+  // [START aiplatform_create_training_pipeline_image_classification]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.\
```

Comment:
no need for the break `\` on the end of the line, for multiline comments.

Reply:
Done.